central/reports/common/report_formatter.go

```diff
@@ -82,7 +82,7 @@ type WatchedImagesResult struct {
 
 // Format takes in the results of vuln report query, converts to CSV and returns zipped CSV data and
 // a flag if the report is empty or not
```

Comment:
Add a comment about the bool field. Without looking at the implementation, there is no way to determine what it is for.

Reply:
I mean, it can be confusing whether true means empty or false means empty because the assumption is based on the var name `found` in the caller. Not a biggie but just a thought.